examples/ctypes-factorial/pixi.toml

```
@@ -0,0 +1,16 @@
+[project]
+name = "pixidemo2"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["liquidcarbon <akscrps@gmail.com>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
+compile = {cmd = "gcc -shared -o src/factorial.so -fPIC src/factorial.c"}
+factorial = {cmd = "python src/factorial.py $1 $2", depends_on = "compile"}
```

```suggestion
factorial = {cmd = "python src/factorial.py, depends_on = "compile"}
```
We're not yet supporting numbered arguments but everything you would add to `pixi run factorial xxx` would be send to that task so in this case you wouldn't need them.

---

Nice!  It's worth calling out in README.  Is there documentation I could reference?  I wasn't able to find this in Pixi or Deno docs, so I just gave it a try, it worked, and I assumed that the `$1 $2` was necessary

---

Added it to the new docs PR : https://github.com/prefix-dev/pixi/pull/435/commits/bf0c0c0454fb443176b12733286db1c495dd49f1

---

Great!  I'll wrap it up later this week.  Thanks for your input!